<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>io_report</class>
 <widget class="QDialog" name="io_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error summary</string>
  </property>
  <property name="windowIcon">
   <iconset resource="highlight-gui.qrc">
    <normaloff>:/highlight.png</normaloff>:/highlight.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Input errors:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listInputErrors"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cbRemoveFiles">
     <property name="text">
      <string>Remove files above from input file list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output errors:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listOutputErrors"/>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Reformatting not possible:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listReformatErrors"/>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Failed syntax tests:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listSyntaxErrors"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="highlight-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>io_report</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>io_report</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
